docs/developers-clear-containers-install.md

```diff
@@ -76,38 +76,26 @@ For more details on the runtime's build system, run:
 $ make help
 ```
 
-4. Qemu-Lite
+4. Qemu-Lite, clear-containers image and kernel
```

Comment:
Because this isn't literal text, we should be consistent in writing "Clear Containers" with capital letters and a space. Could we also make this title a complete sentence, an option:
Qemu-Lite for Clear Containers image and kernel